deploy/kubernetes/helm/che/templates/deployment.yaml

```diff
@@ -72,6 +72,25 @@ spec:
           valueFrom:
             fieldRef:
               fieldPath: metadata.namespace
+
+
+        # If git-self-signed-cert is used then configure Che Server with certificate content
+        # to propagate it to trust store
+        {{- if .Values.global.useGitSelfSignedCerts }}
```

Comment:
I wonder if we can with minimal changes support multiple self-signed certs for different hosts?
On one hand it's quite easy to imagine config map like
```yaml
data:
  my-git.com:9232: --BEGINCERT adssaxxxxxx --
  my-another-git.org:3716: --BEGINCERT adssaxxxxxx --
```
On another hand, it's not clear how to inject each property to the server. Ways I see:
1. Just configure SecretName for Che Server, and then Che Server will ready it from K8s API and propagate to workspaces.
2. Mount whole secret as a folder, where files names are hosts, and content - is ca.cert. And let Che Server know where these folder is mount.